src/mmw/js/src/core/models.js

```diff
@@ -217,8 +217,10 @@ var TaskMessageViewModel = Backbone.Model.extend({
     message: null,
     iconClass: null,
 
-    setError: function() {
-        this.set('message', 'Error');
+    setError: function(err) {
+        var message = err === 'No land cover' ? 'Selected area of interest ' +
```

Comment:
I like the specificity, but I think "...enough land cover data to run the model." shows that's it's a data problem and not a _land cover_ problem, per se.